pkg/controller/elasticsearch/certificates/transport/reconcile_test.go

```diff
@@ -425,3 +464,95 @@ func Test_ensureTransportCertificateSecretExists(t *testing.T) {
 		})
 	}
 }
+
+func Test_mayBeUpdateCAFile(t *testing.T) {
+	type args struct {
+		secret        *corev1.Secret
+		ca            *certificates.CA
+		additionalCAs []byte
+	}
+	tests := []struct {
+		name          string
+		args          args
+		assertSecrets func(t *testing.T, secret map[string][]byte)
+	}{
+		{
+			name: "default settings",
+			args: args{
+				secret:        &corev1.Secret{Data: map[string][]byte{}},
+				ca:            testRSACA,
+				additionalCAs: nil,
+			},
+			assertSecrets: func(t *testing.T, secret map[string][]byte) {
+				t.Helper()
+				assert.Contains(t, secret, "ca.crt")
```

Comment:
Lots of `"ca.crt"` in this file; should we just use the `certificates.CAFileName` in `pkg/controller/common/certificates`?